src/AElf.Kernel.SmartContract.Parallel/ITransactionGrouper.cs

```diff
@@ -6,6 +6,7 @@ namespace AElf.Kernel.SmartContract.Parallel
 {
     public interface ITransactionGrouper
     {
-        Task<(List<List<Transaction>>, List<Transaction>)> GroupAsync(List<Transaction> transactions);
+        Task<(List<List<Transaction>>, List<Transaction>)> GroupAsync(BlockHeader blockHeader,
```

Comment:
do not return Task<(List<List<Transaction>>, List<Transaction>)>,
I can't know what you are returning

Reply:
Replaced tuple with GroupedTransactions object